portfolio.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="portfolio.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.1/css/all.css" />
+    <title>My Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav class="topnav">
+            <div>
+                <a class="mylogo" href="#mylogo">My Logo</a>
+            </div>
+            <ul class="nav__links">
+                <!-- <li class="line"><a href="#portfolio">Portfolio</a></li>
+                <li class="line"><a href="#about">About</a></li>
+                <li class="line"><a href="#contact">Contact</a></li> -->
+                <li class="line"><a href="#portfolio"></a></li>
+                <li class="line"><a href="#about"></a></li>
+                <li class="line"><a href="#contact"></a></li>
+
+            </ul>
+        </nav>
+    </header>
+    <div class="backimage">
+        <div class="welcome">
+            <p>I am Amrendra Glad to see you!</p>
+        </div>
+        <div class="dev-details">I’m a software developer!
+            skilled in React JS, Express JS, Javascript, CSS, HTML, JAVA, Ruby, Ruby on Rails. Hands-on experience in implementing Web
+            Development technologies.
+            Interested in developing softwares.
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate
+            to
+            contact me.
+        </div>
+        <div class="connect-icon">
+            <div class="connect">Lets Connect</div>
+            <div class="social-icons">
+                <a href="https://twitter.com/amrendrak_" target="_blank" rel="noopener noreferrer"><i class="fab fa-twitter"></i></a>
+                <a href="https://www.linkedin.com/in/amrendraakumar/" target="_blank" rel="noopener noreferrer"><i
+                        class="fab fa-linkedin-in"></i></a>
+                <a href="https://github.com/amrendrakind" target="_blank" rel="noopener noreferrer"><i class="fab fa-github"></i></a>
+            </div>
```

Comment:
- [ ] Same goes here, please check the styles in figma and try to match them in your project (lets connect and icons ):

Figma | Yours
------------- | -------------
<img width="377" alt="Screenshot 2022-02-10 at 21 17 44" src="https://user-images.githubusercontent.com/77038610/153461048-9dd5a347-67c9-411b-9841-78b387d4882d.png"> | <img width="374" alt="Screenshot 2022-02-10 at 21 18 16" src="https://user-images.githubusercontent.com/77038610/153461134-c31ec994-634d-4eaf-bb58-df091adb3e1d.png">
 
Please pay special attention to font family and colors 🤗